src/features/members/sagas/detect.saga.ts

```diff
@@ -33,9 +34,18 @@ function* onDetectAction({
                 continue
             }
 
+            const { isMainAccount, isDelegatedAccount } = yield getAccountType(
```

Comment:
I assume an account can be either "delegated" or "main", not both. Can we do what the function name tells us to do and return a type? ;)

```ts
export enum AccountType {
    Main = 'main',
    Delegated = 'delegated'
}

// …

export default function getAccountType(/* … */): AccountType { /* … */ }
```